cloudwatch/src/lambda_function.py

```diff
@@ -77,34 +71,52 @@ def _parse_cloudwatch_log(log, aws_logs_data, log_type):
         pass
 
 
-def _enrich_logs_data(aws_logs_data, context):
-    # type: (dict, 'LambdaContext') -> None
+def _get_additional_logs_data(aws_logs_data, context):
+    # type: (dict, 'LambdaContext') -> dict
+    additional_fields = ['logGroup', 'logStream', 'messageType', 'owner']
+    additional_data = dict((key, aws_logs_data[key]) for key in additional_fields)
+    try:
+        additional_data['function_version'] = context.function_version
+        additional_data['invoked_function_arn'] = context.invoked_function_arn
+    except KeyError:
+        pass
```

Comment:
An informational log could go here, maybe notifying the customer that no such environment variables were found